### PR TITLE
fix: add `jq`, `git`, `unzip` and `curl` to default packages installed

### PR DESCRIPTION
**Pull request by onedr0p**
Hi 👋🏼 

I am using this runner with the [Actions Runner Controller](https://github.com/actions/actions-runner-controller)

These three packages are used by a ton of actions on the marketplace. It would be nice if they were installed and ready for use instead of having to install them with `apt-get` on every single Github workflow. If using more than one job in a workflow you have to have a step to install these in each one.

I know you all probably want to keep a low footprint of installed tools but these are probably just as important as `sudo` for user experience.

### CODE REVIEW

**Comment by withinboredom** [comment]
Will also fix https://github.com/actions/actions-runner-controller/issues/3222

**Review by withinboredom — APPROVED** [review]

**Comment by omri-shilton** [comment]
any update on this?

**Comment by dcardellino24** [comment]
as some people already asked, is there any update on this?

**Comment by onedr0p** [comment]
The silence from the action/runners maintainers is deafening 😭

**Comment by withinboredom** [comment]
There's probably some huge philosophical debate behind closed doors:

- the practical: yeah, this makes sense
- the minimalist: they'd have it be a clone of `scratch` if they could get away with it

**Comment by gcaracuel** [comment]
Can we consider adding `unzip`? Found that used in many actions from the marketplace.

File: images/Dockerfile [hunk 1]
```diff
@@ -41,6 +41,9 @@ ENV ImageOS=ubuntu22
 
 RUN apt-get update -y \
     && apt-get install -y --no-install-recommends \
+    jq \
+    git \
+    curl \
```
Comment (gcaracuel) [inline images/Dockerfile:46]: Can we consider adding `unzip`? Found that used in many actions from the marketplace.

```suggestion
    curl \
    unzip \
```

**Review by joryirving — APPROVED** [review]

**Comment by withinboredom** [comment]
I'm starting to wonder if the project is maintained. I mean, I clearly see them merging PRs but only their own PRs and without much context/comments on them. They don't seem to be interested in community contributions or building this with the community.

**Comment by withinboredom** [comment]
> The decision behind our slim image was to:

> provide a small image that can be used to execute workflows in case you don't need extra tools

Except you can't execute any workflows because not even git is installed... for a <strong>Git</strong>Hub builder...

**Comment by onedr0p** [comment]
This is funny...
https://github.com/actions/runner/pull/3273

**Comment by onedr0p** [comment]
👋🏼 @TingluoHuang

**Comment by omri-shilton** [comment]
any news on this?

**Comment by onedr0p** [comment]
@omri-shilton (or anyone else) if the company you work for has a support contract with GitHub, bring this up in a ticket. Unfortunately it doesn't look like individual contributors PRs are prioritized unless there's a push from a GitHub customer.

**Comment by mbmcmullen27** [comment]
There have been other issues created about how bare the self hosted runner image is. It doesn't seem like a priority for github. Doesn't seem like they're willing to budge on this unfortunately.

https://github.com/actions/runner/issues/3080

I think there was a discussion post about it somewhere too.

**Comment by withinboredom** [comment]
If I had a tinfoil hat 👽 at this point: they want your first jobs to fail so that you don't use self-hosted runners. We all know that a greater barrier to entry means less people will use it. /s

**Comment by nebuk89** [comment]
Sorry for the tumbleweeds, let me poke the team and get someone to look at this.

File: images/Dockerfile [hunk 2]
```diff
@@ -41,12 +41,13 @@ ENV ImageOS=ubuntu22
...
 RUN apt update -y \
-    && apt install -y --no-install-recommends sudo lsb-release gpg-agent software-properties-common \
+    && apt install -y --no-install-recommends sudo lsb-release gpg-agent software-properties-common curl jq unzip \
     && rm -rf /var/lib/apt/lists/*
 
 # Configure git-core/ppa based on guidance here:  https://git-scm.com/download/linux
 RUN add-apt-repository ppa:git-core/ppa \
-    && apt update -y
+    && apt update -y \
+    && apt-get install -y --no-install-recommends git
```
Comment (TingluoHuang) [inline images/Dockerfile:50]:
```suggestion
    && apt install -y --no-install-recommends git
```

**Comment by onedr0p** [comment]
Updated.

**Comment by onedr0p** [comment]
@nebuk89 @TingluoHuang any update?

**Comment by onedr0p** [comment]
I already do that but it wouldn't be required to maintain if the**Git**Hub runner included Git by default.

**Comment by nebuk89** [comment]
I will follow up!

**Review by TingluoHuang — APPROVED** [review]

**Comment by withinboredom** [comment]
🥇

**Comment by GeunSam2** [comment]
I really love this update!!

**Comment by mbelot007** [comment]
When will this change be included in a patch release v2.319.2? Still experiencing issues with this.

**Pull request merged**